src/script/E2EIdentity/E2EIdentityVerification.ts

```diff
@@ -81,17 +89,28 @@ export async function getConversationVerificationState(groupId: string) {
 /**
  * Checks if E2EI has active certificate.
  */
-const fetchSelfDeviceIdentity = async (): Promise<WireIdentity | undefined> => {
-  const conversationState = container.resolve(ConversationState);
-  const selfMLSConversation = conversationState.getSelfMLSConversation();
-  const userIdentities = await getAllGroupUsersIdentities(selfMLSConversation.groupId);
+const getSelfDeviceIdentity = async (): Promise<WireIdentity | undefined> => {
+  let selfMLSConversationGroupId: string;
+  try {
+    // Try to get the self MLS conversation from the conversation state
+    const conversationState = container.resolve(ConversationState);
+    selfMLSConversationGroupId = conversationState.getSelfMLSConversation().groupId;
+  } catch {
```

Comment:
Instead of a try/catch, you can use `conversationState.selfMLSConversation` that will return `undefined` if it is not set

Reply:
Oh right, good idea!

Reply:
done https://github.com/wireapp/wire-webapp/pull/16996/commits/c1b7adec6235da82ca0f7cae79184bfaa51dcb95